### PR TITLE
Add 0.18.0rc1 changelog

### PR DESCRIPTION
**Pull request by fabxc**
@brian-brazil

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+
+* [BUGFIX] Fix operator precedence in PromQL
+* [CHANGE/BUGFIX] Target identity considers path and parameters additionally to host and port
+* [CHANGE] Rename metric `prometheus_local_storage_invalid_preload_requests_total` to `prometheus_local_storage_non_existent_series_matches_total`
+* [FEATURE] Deduplicate targets within the same scrape job
+* [FEATURE] Add varbit chunk encoding (higher compression, more CPU usage – disabled by default)
+* [FEATURE] Add `holt_winters` query function
+* [FEATURE] Add relative complement `unless` operator to PromQL
+* [ENHANCEMENT] Handle data corruption more gracefully
+* [ENHANCEMENT] Quarantine series file on write issues
```
Comment (beorn7) [inline CHANGELOG.md]: I think these two items are essentially the same (or the latter is the way to do the former).

Comment (beorn7) [inline CHANGELOG.md]: Perhaps: "Quarantine series file if data corruption is encountered (instead of crashing)."

**Comment by beorn7** [comment]
I assume we will add same more instructions to the announcement mail (like how to activate the varbit encoding and such).

**Comment by fabxc** [comment]
Yes. What's "and such"? I don't see anything else configuration wise – PromQL changes are found in the documentation, which we can link. We didn't do this before though.

**Comment by brian-brazil** [comment]
:+1:

**Comment by beorn7** [comment]
"and such" could be the explanation about the improvement in query time if you have a lot of series churn (so that people can do the trade off if they want to upgrade or not - for those in less agile environment where an upgrade is a big deal).

File: CHANGELOG.md (hunk 1 above)
Comment (beorn7) [inline CHANGELOG.md]: Or "instead of panic" if we want to use the Go lingo.

**Comment by fabxc** [comment]
Can you give me a short paragraph to include here?

**Comment by grobie** [comment]
Please publish a blog post about the release and all the necessary information (explanations, links, howtos, etc.). The mailing list is nice but not widely used.

**Comment by beorn7** [comment]
I'll phrase a paragraph about the chunk encoding and the index lookups once I'm out of the meeting I'm currently in

**Comment by beorn7** [comment]
@fabxc Suggestion:

This release introduces a new chunk encoding with variable bit width, in parts inspired by [Facebook's "Gorilla" paper](http://www.vldb.org/pvldb/vol8/p1816-teller.pdf). At the price of increased CPU cost and run-time of queries, the chunk space requirement drops by ~60% for typical data sets. The varbit encoding has to be enabled with the command line flag `-storage.local.chunk-encoding-version=2`, all _newly created_ chunks will have the new chunk encoding. It is safe to use a mix of different chunk encodings.

Several storage improvements dramatically increase the performance of instant queries for the most recent value of time series, as it typically happens in rule evaluations and for federation. On servers with a lot of series churn (i.e. many older series that don't receive samples anymore), we have seen a 12x speedup of rule evaluations with CPU usage reduced by 75%.

**Comment by fabxc** [comment]
Great numbers. Really nice work!

**Pull request merged**